### DISCUSSION TITLE
V3 `wire:navigate` should scroll to top

### DISCUSSION THREAD

**Discussion by bilogic** [livewire/livewire#6117] [7. Bugs (archived)]
### Livewire version

3

### Which PHP version are you using?

PHP 8.1

### Steps To Reproduce

`wire:navigate` works and is loading the new page quickly. However, if I was in the middle of the page, I will also be in the middle of the newly loaded page. But this differs from how the browser handles it, which is to start from the top of the new page

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment]
@bilogic yeah agreed, will see what Caleb thinks of that.

**Comment by calebporzio** [comment]
Hmmm I was unable to reproduce. Everything seems to work well.

I added a passing test in this commit to demonstrate: https://github.com/livewire/livewire/commit/5ec6a38965dd5233e6295c418ab1c689314c956c

If you can provide a failing test, we can further assist. Thanks!

**Comment by bilogic** [comment]
I found the issue on my side, it was caused by a stray JS. Thanks!

**Reply by robsontenorio** [reply]
@bilogic could you please share your solution? I have same issue, but can’t spot the problem.
